content/code_samples/escrow/websocket/account_objects-response.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
account_objects-response.json replaces account_objects_response_sender.json and serves as the response code sample for both sender and receiver.